### DISCUSSION TITLE
Outlet navigation

### DISCUSSION THREAD

**Discussion by Venefilyn** [JesusTheHun/storybook-addon-remix-react-router#37] [Q&A] [answered]
When it comes to `reactRouter.outlet`, what is the recommended way of handling navigation between different tabs that changes the Outlet results? In this case, `Jobs` component has navigation tabs between different tables, but `outlet` option only allows for one entry

![image](https://github.com/JesusTheHun/storybook-addon-react-router-v6/assets/6598829/fa86269b-83b7-40d2-b057-d7fb96b93cc6)

https://github.com/JesusTheHun/storybook-addon-react-router-v6/assets/6598829/326b90f5-7df7-4ca5-944b-b73de193f954

**Comment by JesusTheHun** [comment, marked as answer]
Currently there is no built-in way to handle multiple outlet.
You must create a story per outlet.

I'm working on a v2 that will support multiple outlets

**Reply by Venefilyn** [reply]
Gotcha, thanks for the update!

**Comment by JesusTheHun** [comment]
@SpyTec Hi 👋
This is now supported out of the box : https://github.com/JesusTheHun/storybook-addon-react-router-v6/releases/tag/v2.0.0-rc.0
